cpp/llm_chat.cc

```diff
@@ -354,7 +379,13 @@ class LLMChat {
     if (this->max_window_size_ == -1) {
       this->max_window_size_ = std::numeric_limits<int64_t>::max();
     }
-    this->conversation_ = Conversation::FromTemplate(conv_template);
+    // The RWKV World series of models have different prompts compared to other models like RWKV Raven.
+    if(containsBothWords(this->model_name_)){
+      this->conversation_ = Conversation::FromTemplate("rwkv-world");
+    }
+    else{
+      this->conversation_ = Conversation::FromTemplate(conv_template);
```

Comment:
Please consider moving the logic to 

https://github.com/mlc-ai/mlc-llm/blob/678939c594b2b6593c7769896d0b4d00447f0055/cpp/conv_templates.cc#L529

Reply:
ok

Reply:
done